.github/workflows/cypress-stress.yaml

```diff
@@ -0,0 +1,37 @@
+# Triggers weekly on a Friday morning at 05:30 UTC
+name: Cypress Stress Tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 5 * * Fri'
+jobs:
+  tests:
+    name: "Run Stress Tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run Cypress tests
+        uses: cypress-io/github-action@v4.1.1
+        with:
+          browser: chrome
+          start: npx serve -p 8080 webapp
```

Comment:
Where is `webapp` coming from?

Reply:
Ha, I forgot I need to get element-web from somewhere.